variables.tf

```diff
@@ -13,8 +13,43 @@ variable "azure_region" {
   description = "Region of azure storage"
 }
 
-# TODO: Make as much autogenerated as possible and then document the rest of these
+variable "kubernetes_namespace" {
+  default     = "taskcluster"
+  type        = "string"
+  description = "Optional namespace to run services in"
+}
 
-variable "auth_pulse_username" {}
+variable "root_url" {
+  type        = "string"
+  description = "Taskcluster rootUrl"
+}
 
-variable "auth_pulse_password" {}
+variable "tls_crt" {
+  type        = "string"
+  description = "Taskcluster tls certificate"
```

Comment:
worth mentioning this is for the rootUrl

Reply:
Ah yeah. I have a big TODO to update all of the documentation stuff at some point. It is very lacking currently.